h2integrate/core/test/test_finances.py

```diff
@@ -67,3 +67,105 @@ def test_electrolyzer_refurb_results():
+            "taxable": True,
+        },
+        "incidental revenue": {"value": 0.0, "escalation": 0.0},
+        "credit card fees": 0,
+        "sales tax": 0.07,
+        "road tax": {"value": 0.0, "escalation": 0.0},
+        "labor": {"value": 0.0, "rate": 0.0, "escalation": 0.0},
+        "maintenance": {"value": 0, "escalation": 0.02},
+        "rent": {"value": 0, "escalation": 0.02},
+        "license and permit": {"value": 0, "escalation": 0.02},
+        "non depr assets": 0.0,
+        "end of proj sale non depr assets": 0.0,
+        "installation cost": {
+            "value": 0,
+            "depr type": "Straight line",
+            "depr period": 4,
+            "depreciable": False,
+        },
+        "one time cap inct": {
+            "value": 0.0,
+            "depr type": "MACRS",
+            "depr period": 3,
+            "depreciable": True,
+        },
+        "property tax and insurance": 0.015,
+        "admin expense": 0.03,
+        "tax loss carry forward years": 0,
+        "capital gains tax rate": 0.15,
+        "tax losses monetized": True,
+        "sell undepreciated cap": True,
+        "loan period if used": 10,
+        "debt equity ratio of initial financing": 2.3333333333333335,
+        "debt interest rate": 0.05,
+        "debt type": "Revolving debt",
+        "total income tax rate": 0.21,
+        "cash onhand": 6,
+        "general inflation rate": 0.02,
+        "leverage after tax nominal discount rate": 0.08,
+    }
+    plant_config = {
+        "finance_parameters": {
+            "costing_general_inflation": 0.0,
+            "pf_params": {"params": pf_params},
+            "depreciation_method": "Straight line",
+            "depreciation_period": 20,
+            "depreciation_period_electrolyzer": 10,
+        },
+        "plant": {
+            "atb_year": 2022,
+            "plant_life": 30,
+            "installation_time": 24,
+            "cost_year": 2022,
+            "grid_connection": True,
+            "ppa_price": 0.05,
+        },
+        "policy_parameters": {
+            "electricity_itc": 0.3,
+            "h2_storage_itc": 0.3,
+            "electricity_ptc": 25,
+            "h2_ptc": 3,
+        },
+    }
+
+    tech_config = {
+        "electrolyzer": {
+            "model_inputs": {
+                "financial_parameters": {
+                    "replacement_cost_percent": 0.1,
+                }
+            }
+        },
+    }
+
+    prob = om.Problem()
+    comp = ProFastComp(plant_config=plant_config, tech_config=tech_config)
+    prob.model.add_subsystem("comp", comp, promotes=["*"])
+
+    prob.setup()
+
+    prob.set_val("capex_adjusted_electrolyzer", 1.0e7, units="USD")
+    prob.set_val("opex_adjusted_electrolyzer", 1.0e4, units="USD/year")
+
+    prob.set_val("total_hydrogen_produced", 4.0e5, units="kg/year")
+    prob.set_val("time_until_replacement", 5.0e3, units="h")
+
+    prob.run_model()
+
+    assert prob["LCOH"] == approx(4.27529137)
```

Comment:
Thanks for putting in a test. Can you include a description of what this test is based on? Just saying it is based on output at time of writing is sufficient, but it helps future devs know how seriously to take changes and if the changed assumptions and/or results should be compared to some other source.